app/assets/json/translations/react/tl.json

```diff
@@ -1,6 +1,17 @@
 {
   "config.routePrefix": "tl",
   "footer.cityCountyOfSf": "Lungsod at County ng San Francisco",
+  "footer.contact": "Kontakin ",
```

Comment:
nit: extra space here. Looks like it was in the original locale.tl file but we might as well fix it here